api/conf/schema.json

```diff
@@ -4411,6 +4519,22 @@
 														"required": ["connect", "read", "send"],
 														"type": "object"
 													},
+													"tls": {
+														"properties": {
+															"client_cert": {
+																"maxLength": 65536,
+																"minLength": 128,
+																"type": "string"
+															},
+															"client_key": {
+																"maxLength": 65536,
+																"minLength": 128,
+																"type": "string"
+															}
+														},
+														"required": ["client_cert", "client_key"],
+														"type": "object"
+													},
```

Comment:
The style seems not so good, while this could be fixed in other PR. Thanks.

Reply:
I don’t know why it appears like this, it may be squeezed. 
You could see that the display on the `Conversation` page is normal.

And it’s OK whether it’s on my computer or in github:
https://github.com/nic-chen/incubator-apisix-dashboard/blob/json-schema-sync/api/conf/schema.json#L4522-L4537

Reply:
> I don’t know why it appears like this, it may be squeezed.
> You could see that the display on the `Conversation` page is normal.
> 
> And it’s OK whether it’s on my computer or in github:
> https://github.com/nic-chen/incubator-apisix-dashboard/blob/json-schema-sync/api/conf/schema.json#L4522-L4537

@nic-chen ok, got it. It's ok actually. Thanks.